controllers/util/provider.go

```diff
@@ -71,6 +76,13 @@ type AzureCredentials struct {
 	ARMTenantID       string `yaml:"armTenantID"`
 }
 
+type VSphereCredentials struct {
+	VSphereUser               string `yaml:"vSphereUser"`
+	VSpherePassword           string `yaml:"vSpherePassword"`
+	VSphereServer             string `yaml:"vSphereServer"`
+	VSphereAllowUnverifiedSSL string `yaml:"vSphereAllowUnverifiedSSL"`
```

Comment:
How about marking it as optional?

Reply:
How should I do it, such as marking it as an optional ？

Reply:
Like `yaml:"vSphereAllowUnverifiedSSL,omitempty"`.

Reply:
ok, thank you!